tests/conversion__array.js

```diff
@@ -51,6 +51,20 @@ describe('OUT-array (zero-terminated)', () => {
   expect(alternates, [ 'special' ])
 })
 
+describe('OUT-array (zero-terminated, of guint8)', () => {
+  const cmd = 'echo foo';
+
+  const result = glib.spawnCommandLineSync(cmd);
+
+  // @standard_output: (out) (array zero-terminated=1) (element-type guint8) (optional): return location for child output
+  // Because (element-type guint8), the result becomes array of numbers, not string.
+  const [success, standard_output] = result;
+  const expected_stdout = Array.from('foo\n').map( c => c.codePointAt(0) );
+
+  expect(success, true);
+  expect(standard_output, expected_stdout);
+});
+
```

Comment:
Lint: JS conventions for this project:
 - `camelCase` variables
 - no semi-colons
 - no space inside `.map()` parenthesis

Reply:
Fixed.